scripts/quests/windurst/Heaven_Cent.lua

```diff
@@ -0,0 +1,274 @@
+-----------------------------------
+-- Heaven Cent
+-----------------------------------
+-- !addquest 2 49
+-- Mashuu-Ajuu 130 -5 167
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+-----------------------------------
+ID = require('scripts/zones/Maze_of_Shakhrami/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.HEAVEN_CENT)
+
+local incorrect =
+{
+    ID.text.SHIVA_WEST,
+    ID.text.LEAVIATHAN_SOUTH,
+    ID.text.TITAN_NORTH,
+    ID.text.ODIN_EAST,
+}
+
+local correct =
+{
+    ID.text.ALEXANDER_NE,
+    ID.text.SHIVA_EAST,
+    ID.text.ODIN_NORTH,
+    ID.text.IFRIT_NW,
+}
+
+quest.reward =
+{
+    gil = 4800,
+    title = xi.title.NIGHT_SKY_NAVIGATOR,
+    fame = 100,
+    fameArea = xi.quest.fame_area.WINDURST,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getFameLevel(xi.quest.fame_area.WINDURST) >= 5
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Ropunono'] = quest:progressEvent(284),
+
+            onEventFinish =
+            {
+                [284] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Ropunono'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(289, 0, 557, 545, 0, 0, 4095, 4)
```

Comment:
Are the params passed here key items or items? We could replace these with their actual names for easier readability if so.

Reply:
Ugh I GUESS